src/components/ContactFooter.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Section, SectionTitle, TextWrapper } from 'components/Section';
+
+const ContactFooter = () => {
+  return (
+    <Footer>
+      <SectionTitle>CONTACT</SectionTitle>
+      <FooterWrap>
+        <FooterContainer>
+          <h2>Cecilia Frånlund</h2>
+          <h2>jr frontend developer</h2>
+          <h2>ceciliafraanlund@gmail.com</h2>
+          <h2>+46 728 5454 14</h2>
+          <h2>Stockholm, Sweden</h2>
+        </FooterContainer>
+      </FooterWrap>
+    </Footer>
+  );
+};
+export default ContactFooter;
+
+const FooterWrap = styled(TextWrapper)`
```

Comment:
I did not know that you can use a styled component in another styled component like this, with the parentheses. Cool! Thanks for this learning!